src/app/containers/IndexPageSection/utilities/filterAllowedItems.test.jsx

```diff
@@ -147,7 +147,7 @@ const allowedItemsTest = value => {
       const isFirstSection = true;
       expect(
         getAllowedItems({ items, isFirstSection, showAllPromos }),
-      ).toStrictEqual(numberOfStories[value].expectedFrontSectionAllowedItems);
+      ).toStrictEqual(numberOfStories[value].expectedShowAllPromosAllowedItems);
```

Comment:
what has triggered this to change?

Reply:
The requirements of the ticket. We want to show show all promos if it's the 1st section and showAllPromos is set to true.